src/golang/cmd/server/handler/edit_workflow.go

```diff
@@ -126,6 +135,39 @@ func (h *EditWorkflowHandler) Perform(ctx context.Context, interfaceArgs interfa
 	}
 	defer database.TxnRollbackIgnoreErr(ctx, txn)
 
+	dag, err := utils.ReadLatestDAGFromDatabase(
+		ctx,
+		args.workflowId,
+		h.WorkflowRepo,
+		h.DAGRepo,
+		h.OperatorRepo,
+		h.ArtifactRepo,
+		h.DAGEdgeRepo,
+		txn,
+	)
+	if err != nil {
+		return nil, http.StatusInternalServerError, errors.Wrap(err, "Unable to update workflow.")
+	}
+
+	// Schedule validation needs to happen inside the `txn` to prevent
```

Comment:
Where is the `txn` you are referring to?

Reply:
Addressed